### PR TITLE
Update XFCE related templates

### PR DESCRIPTION
**Pull request by fepitre**
@andrewdavidwong There is certainly things to fix. I let this open and finish it soon.

### CODE REVIEW

File: user/managing-os/xfce-templates.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+title: Xfce TemplateVMs
+permalink: /doc/templates/xfce/
+redirect_from:
+- /doc/templates/xfce/
+- /doc/xfce/
+- /en/doc/templates/xfce/
+- /doc/Templates/Xfce/
+- /wiki/Templates/Xfce/
+---
+
```
Comment (fepitre) [inline user/managing-os/xfce-templates.md:12]: @andrewdavidwong I've read the modifications do you see anything wrong with the above modifications of Fedora Xfce into a more general cathegory like the "Minimal" one?

Comment (andrewdavidwong) [reply user/managing-os/xfce-templates.md:12]: I was thinking the same thing. Sounds like a good move to me. :slightly_smiling_face:

**Comment by andrewdavidwong** [comment]
Let me know when this PR is ready, @fepitre.

**Comment by fepitre** [comment]
@andrewdavidwong It's good you can merge. Thank your :)

**Pull request merged**